### ISSUE TITLE
Need for a `.deb` package has got more urgent with Ubuntu Noble 

### ISSUE THREAD

**Issue by lproven** [BrowserWorks/waterfox#3467]
I have requested a native Debian package before. 

Previously this need was fulfilled by a 3rd party, called Hawkeye, who published his own repo -- and you were very lucky to have that help.

https://build.opensuse.org/project/show/home:hawkeye116477:waterfox

However...

I contacted him & he doesn't use Waterfox any more, and does not plan to maintain the repo.

Ubuntu 24.04 uses a new format for Debian repository info, called DEB822 format:

https://repolib.readthedocs.io/en/latest/deb822-format.html

His repo has not been updated since Ubuntu 23.10. 

It no longer works for me on Ubuntu 24.04. 

Even Mozilla now distributes .deb packages of Firefox. 

It is time. Fix this.

The Debian family of distros have 10x the number of users of all .RPM formats put together. Ignore Red Hat for now: it's a rounding error, just a wealthy one.

**Comment by LoopControl** [comment]
Yep, was looking to move away from FF so came here only to see no .deb/Ubuntu package.

Going to try Librewolf now since they have .debs

Edit: Worked perfectly. For anyone on Ubuntu, I'd recommend using that if you need .deb

**Comment by hawkeye116477** [comment]
In case of Ubuntu 24.04, it's building, but failed. There is some error and I didn't figured out for now the cause.
`MESA: error: ZINK: vkCreateInstance failed (VK_ERROR_INCOMPATIBLE_DRIVER)` (crash during profile initialization). 
It might be resumed when I'll resolve or someone else will help. I'll probably keep maintaining it until G6 EOL and then maybe only AppImage will remain.

**Comment by lproven** [comment]
> There is some error and I didn't figured out for now the cause.

Thanks for the update, and thanks for the packages! I have been using them for years now, on multiple PCs and distros, and they have been a real life-saver.

**Comment by hawkeye116477** [comment]
There is good news, I found fix and repository is now live.

**Comment by drax-xard** [comment]
I second this request. I'm trying to migrate from FF after the Terms of Use debacle but I don't want to depend on 3rd party builds. Thanks!

**Comment by MrAlex94** [comment]
https://download.opensuse.org/repositories/isv:/BrowserWorks/
